src/rollback_to_stable/rts_api.c

```diff
@@ -195,6 +195,9 @@ __rollback_to_stable(WT_SESSION_IMPL *session, const char *cfg[], bool no_ckpt)
     WT_STAT_CONN_SET(session, txn_rollback_to_stable_running, 1);
     WT_WITH_CHECKPOINT_LOCK(
       session, WT_WITH_SCHEMA_LOCK(session, ret = __rollback_to_stable_int(session, no_ckpt)));
+
+    __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
```

Comment:
Would it make sense to add a similar message at around line 183 indicating that RTS is starting?

Reply:
On line 77 of rts_api.c there is a message which says: "performing rollback to stable with stable_timestamp=" which I think should suffice. @kommiharibabu would you agree?

Reply:
It looks something like this, which is from one of the unit tests that failed on the ticket:
```
[2023/03/17 03:21:03.856] [1679023257:917968][33448:0x7f699a135800], test_rollback_to_stable25.test_rollback_to_stable25.test_rollback_to_stable25(10l.upd.20u.nil.30l.del.chk20.roll15), txn rollback_to_stable: [WT_VERB_RTS][DEBUG_1]: [INIT] performing rollback to stable with stable_timestamp=(0, 15) and oldest_timestamp=(0, 2)
[2023/03/17 03:21:03.856] [1679023257:918009][33448:0x7f699a135800], test_rollback_to_stable25.test_rollback_to_stable25.test_rollback_to_stable25(10l.upd.20u.nil.30l.del.chk20.roll15), txn rollback_to_stable: [WT_VERB_RTS][DEBUG_1]: [TREE_SKIP] file:rollback_to_stable25.wt: tree skipped with durable_timestamp=(0, 10) and stable_timestamp=(0, 15) or txnid=6
[2023/03/17 03:21:03.856] [1679023257:918024][33448:0x7f699a135800], test_rollback_to_stable25.test_rollback_to_stable25.test_rollback_to_stable25(10l.upd.20u.nil.30l.del.chk20.roll15), txn rollback_to_stable: [WT_VERB_RTS][DEBUG_1]: [END] rollback to stable finished
```

Should I add a comment explaining why we don't have a start message?

Reply:
Why not just add the word "Start" so that it reads "Start performing rollback..." and it is very clear.

Reply:
Yes, the existing verbose message with more details is good as the start message. I feel there is no need to add a new start message.

Reply:
I'll add the word "Start" as Sue suggested.